### ISSUE TITLE
install no scale cluster-proxy replicas

### ISSUE THREAD

**Issue by chris93111** [open-cluster-management-io/cluster-proxy#105]
Hi 
With fresh installation of cluster-proxy addon, the status of ManagedProxyConfiguration return a successfull deployment with 3 replicas in spec but 0 is set in status , so is needed to scale manually the deployment

```
spec:
  authentication:
    dump:
      secrets:
        signingAgentServerSecretName: agent-server
        signingProxyClientSecretName: proxy-client
        signingProxyServerSecretName: proxy-server
    signer:
      type: SelfSigned
  proxyAgent:
    image: 'quay.io/open-cluster-management/cluster-proxy:v0.1.4'
    replicas: 3
  proxyServer:
    entrypoint:
      port: 8091
      type: PortForward
    image: 'quay.io/open-cluster-management/cluster-proxy:v0.1.4'
    inClusterServiceName: proxy-entrypoint
    namespace: open-cluster-management-addon
status:
  conditions:
    - lastTransitionTime: '2022-05-03T19:36:47Z'
      message: 'Replicas: 0'
      reason: SuccessfullyDeployed
      status: 'True'
      type: ProxyServerDeployed
```

**Comment by yue9944882** [comment]
this should be fixed in v0.2.0. can you upgrade using the latest release from the helm registry https://github.com/open-cluster-management-io/cluster-proxy/pull/107?

**Comment by xuezhaojun** [comment]
/close
Since the issue is fixed.
